### DISCUSSION TITLE
Official RetroAchievements Mobile App

### DISCUSSION THREAD

**Discussion by rabbids4eva** [RetroAchievements/RAWeb#736] [Ideas]
I saw there was an unofficial app on Google Play that lets you keep track of what's new on RA. It's kinda limited, so why not make an official RA app.

**Comment by meleu** [comment]
Our main goal is to make the site so mobile friendly that people won't see the need to use an external app. But, well, anyone is free to create such app. What we can do to help it is to improve the API, and it's exactly the subject of #186

**Comment by CoolCanuck** [comment]
Apps are a kinda past their time. 

Progressive Web Apps and Chat Bots are becoming more used :).
